e2e/preference/create.spec.ts

```diff
@@ -0,0 +1,58 @@
+import type { PreferenceCreateData } from '@src/clients/preference/create/types';
+import MercadoPago, { Preference } from '@src/index';
+import { config } from '../e2e.config';
+
+describe('Preference IT, create', () => {
+	test('should create a request with success', async () => {
+
+		const client = new MercadoPago({  accessToken: config.access_token, options: { timeout: 5000 } });
+		const preference = new Preference(client);
+
+		const preferenceRequest: PreferenceCreateData = {
+			body: {
+				items: [
+					{
+						'id': '4567',
```

Comment:
Objects do not have to have quotation marks on both sides, only quotation marks in the string, that is, in place of `{'id': '123'}` it must be `{id: '123'}`